pilz_store_positions/src/pilz_store_positions/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2019 Pilz GmbH & Co. KG
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from .ros_message_serializer import RosMessageSerializer
+from .pose_file_tf_publisher import PoseFileTFPublisher
```

Comment:
Maybe `import *` as in [pilz_robot_programming](https://github.com/PilzDE/pilz_industrial_motion/blob/kinetic-devel/pilz_robot_programming/src/pilz_robot_programming/__init__.py)?  or [tf2_ros](https://github.com/ros/geometry2/blob/noetic-devel/tf2_py/src/tf2_py/__init__.py)

Reply:
If there are multiple classes to be imported, `*` makes sense, yes. In this case we only need one class per file so I think is it is cleaner to import them named.